dowhy/timeseries/timeseries.py

```diff
@@ -0,0 +1,82 @@
+import networkx as nx
+import pandas as pd
+import matplotlib.pyplot as plt
+
+def create_graph_from_user():
+    # Initialize an empty directed graph
+    graph = nx.DiGraph()
+    
+    # Instructions for the user
+    print("Enter the graph as a list of edges with time lags. Enter 'done' when you are finished.")
+    print("Each edge should be entered in the format 'node1 node2 time_lag'. For example: 'A B 4'")
+    
+    # Loop to receive user input
+    while True:
+        edge = input("Enter an edge: ")
+        if edge.lower() == "done":
+            break
+        edge = edge.split()
+        if len(edge) != 3:
+            print("Invalid edge. Please enter an edge in the format 'node1 node2 time_lag'.")
+            continue
+        node1, node2, time_lag = edge
+        try:
+            time_lag = float(time_lag)
+        except ValueError:
+            print("Invalid weight. Please enter a numerical value for the time_lag.")
+            continue
+        graph.add_edge(node1, node2, time_lag=time_lag)
+    
+    return graph
+
+def create_graph_from_csv(file_path):
+    # Read the CSV file into a DataFrame
+    df = pd.read_csv(file_path)
+    
+    # Initialize an empty directed graph
+    graph = nx.DiGraph()
+    
+    # Add edges with weights to the graph
+    for index, row in df.iterrows():
+        graph.add_edge(row['node1'], row['node2'], time_lag=row['time_lag'])
```

Comment:
It appears that this assumes a very specific CSV content/schema. Can you add a docstring with an example?